source/one-event.ts

```diff
@@ -4,12 +4,13 @@ type RemovableEvent<T = (...args: unknown[]) => unknown> = {
 };
 
 export async function oneEvent<
-	Event extends RemovableEvent,
-	Filter extends (..._arguments: any[]) => boolean,
+	Event extends RemovableEvent<(...args: any[]) => void>,
+	EventParameters extends Parameters<Parameters<Event['addListener']>[0]>[0],
```

Comment:
I think the final `[0]` is still picking only one parameter. CI-only tests are failing. We need to add the two tests I suggested:

- one in the demo extension (you can copy-paste mine)
- one in oneEvent.test.ts

Reply:
Sorry forget it. That push was missed